suite_cam4.xml

```diff
@@ -0,0 +1,204 @@
+      <scheme>diag_export</scheme>                    <!-- Output various host model surface fluxes to history file -->
+    </time_split>
+  </group>
+  <group name="physics_ac">
+    <!--cam6:  diffusion=HB, chem=none -->
+    <time_split>
+      <scheme>flux_avg_run</scheme>                   <!-- Smooth/average surface fluxes, but only if phys_do_flux_avg is True -->
+      <scheme>physics_state_check</scheme>            <!-- Only if state_debug_checks is True. This scheme only looks for infs/nans, and kills the model if found -->
+      <scheme>calc_flx_net</scheme>                   <!-- Accumulate net surface fluxes, which is necessary for spectral dycores -->
+      <!-- Negative moisture/tracer check -->
+      <!-- **************************** -->
+      <scheme>qneg4</scheme>                          <!-- Modify surface moisture and heat fluxes to prevent negative moisture values, and output diagnostics -->
+      <scheme>aoa_tracers_timestep_tend</scheme>      <!-- Calculate age of air tracer tendencies.  Also output tendencies to history file -->
+      <scheme>check_tracers_chng</scheme>             <!-- Global tracer mass checker that kills model if error is large enough. -->
+      <scheme>co2_cycle_set_ptend</scheme>            <!-- Calculate CO2 tendencies.  Only matters if co2_flag or co2_readFlux_aircraft is True -->
+      <!-- **************************** -->
+      <!-- Vertical diffusion -->
+      <!-- **************************** -->
+      <scheme>set_dry_to_wet</scheme>                 <!-- Convert mixing ratios from dry air to air + water vapor -->
+      <scheme>tint_calc</scheme>                      <!-- Interpolate air temperature to layer interface levels -->
+      <scheme>ubc_get_vals</scheme>                   <!-- Calcuate upper-level boundary conditions (depends on chemistry and model-top choice) -->
+      <scheme>set_top_tint_val</scheme>               <!-- Calculate air temperature at top interface level -->
+      <scheme>rhoi_calc</scheme>                      <!-- Interpolate air density and dry air density to layer interface levels -->
+      <scheme>compute_tms</scheme>                    <!-- Calculate Turbulent Mountain Stress (TMS) tendency -->
+      <scheme>tms_diag_output</scheme>                <!-- Calculate TMS diangostic variables, and output them to history file -->
+      <scheme>compute_blj</scheme>                    <!-- Calculate Sub-Grid Orgraphic (SGO) drag via Beljaars parameterization -->
+      <scheme>blj_diag_output</scheme>                <!-- Calculate SGO diagnostic variables, and output them to history file -->
+      <scheme>theta_calc</scheme>                     <!-- Calculate potential temperature (managed by registry/framework?) -->
+      <scheme>virtem</scheme>                         <!-- Calculate virtual temperature (managed by registry/framework?) -->
```

Comment:
This one could potentially be added but first, we have to figure out how to define 'wet'.

Reply:
I have gone ahead and left the comment as-is for now, simply because I think finding a definition for "wet" would be beneficial, as there are numerous other variables (including different versions of potential temperature) used in CAM6 that would benefit from it as well.